fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -1181,7 +1181,7 @@ public void subscribeToLao(String laoId) {
                 lao -> {
                   Log.d(TAG, "got an update for lao: " + lao.getName());
                   mCurrentLao.postValue(lao);
-                  boolean isOrganizer = lao.getOrganizer().equals(keyManager.getMainPublicKey());
+                  boolean isOrganizer = keyManager.getMainPublicKey().equals(lao.getOrganizer());
```

Comment:
What does this affect ? lao.getOrganizer() should not be null anyway.

Reply:
Well this was the reason it crashed instead of gracefully, just waiting for the catchup to complete. Now it cannot be a problem anymore so it's not needed.

Reply:
Oh ! I see. Ok